internal/collector/factories.go

```
@@ -74,26 +51,29 @@ func OTelComponentFactories() (otelcol.Factories, error) {
 		Exporters:  exporters,
 	}
 
-	return factories, errs
+	return factories, nil
```

Do we need to return an error if it is always nil ?

---

yeah this struct requires it https://pkg.go.dev/go.opentelemetry.io/collector/otelcol#CollectorSettings